src/imgtools/io/common.py

```
@@ -1,41 +1,8 @@
 import os
 import pathlib
-from typing import Dict
+from typing import Dict, Generator
```

Ruff (0.8.0)

`typing.Generator` imported but unused

Remove unused import: `typing.Generator`

(F401)